common/test/java/com/couchbase/lite/internal/core/C4BaseTest.java

```diff
@@ -115,6 +118,30 @@ protected void createRev(C4Database db, String docID, String revID, byte[] body)
         createRev(db, docID, revID, body, 0);
     }
 
+
+    // This method is a mock for c4Collection_putDoc. It checks whether a document is updated with a new revision id,
```

Comment:
Cool.  Should we remove this when c4Collection_putDoc goes in?  ... or will it still be useful for testing?

Reply:
We can remove this when putDoc goes in

Reply:
Great... Stick that in a comment, I think....